.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: 💡 Feature Request
+    url: https://github.com/orgs/polarsource/discussions/new?category=feature-requests
+    about: Please submit feature requests here.
+  - name: 🛟 Improvement Request
```

Comment:
This is a bit redundant with Feature Request. I would prefer to have a support link to our Discord. Something like:

```yml
  - name: 🛟 Support
    url: https://discord.com/invite/STfRufb32V
    about: Join us on Discord to ask support questions
```